rules/windows/process_creation/proc_creation_win_susp_lolbin_non_c_drive.yml

```
@@ -0,0 +1,29 @@
+title: Wscript Execution from Non C Drive
+id: 5b80cf53-3a46-4adc-960b-05ec19348d74
+status: experimental
+description: Detects Wscript or Cscript executing from a drive other than C. This has been observed with Qakbot executing from within a mounted ISO file.
+author: Aaron Herman
+date: 2022/10/01
+references:
+    - https://github.com/pr0xylife/Qakbot/blob/main/Qakbot_BB_30.09.2022.txt
+tags:
+    - attack.execution
+    - attack.t1059
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection_lolbin:
+        CommandLine|contains:
+            - 'wscript.exe'
+            - 'cscript.exe'
+    selection_exetensions:
+        CommandLine|contains: '.js'
+    selection_drive_path:
+        CommandLine|contains: ':\'
+    filter_drive_path:
+        CommandLine|contains: 'C:\'
```

I think we would also have to exclude command lines that use environment variables like %UserProfile% or %ProgramData%. Maybe it's easier to just list all kinds of drive names. I also wonder what happens when someone executes scripts from network drives during startup or the like. There would be many FPs. 

```suggestion
    filter_drive_path:
        CommandLine|contains: 'C:\'
    filter_env_vars:
        CommandLine|contains: '%'
```

---

Added `%`